easybuild/easyconfigs/a/ARTS/ARTS-2.2.64-gompi-2019a.eb

```
@@ -0,0 +1,54 @@
+# Authors:: Jack Perdue <j-perdue@tamu.edu> - TAMU HPRC - https://hprc.tamu.edu
+
+easyblock = 'CMakeMake'
+
+name = 'ARTS'
+version = '2.2.64'
+
+homepage = 'http://www.radiativetransfer.org/'
+
+description = """
+ ARTS is a radiative transfer model for the millimeter and sub-millimeter
+ spectral range. There are a number of models mostly developed explicitly
+ for the different sensors.
+"""
+
+toolchain = {'name': 'gompi', 'version': '2019a'}
+toolchainopts = {'opt': True, 'cstd': 'gnu++98', 'openmp': True}
+
+source_urls = ['http://www.radiativetransfer.org/misc/download/stable/2.2/']
```

@JackPerdue We should use `%(version_major_minor)s` here rather than hardcoding the `2.2`?

```suggestion
source_urls = ['http://www.radiativetransfer.org/misc/download/stable/%(version_major_minor)s/']